GliaWidgets/ViewModel/Chat/Unread/UnreadMessagesHandler.swift

```diff
@@ -2,34 +2,47 @@ class UnreadMessagesHandler {
     private let unreadMessages: ObservableValue<Int>
     private let isWindowVisible: ObservableValue<Bool>
     private let isViewVisible: ObservableValue<Bool>
+    private let isChatScrolledDown: ObservableValue<Bool>
```

Comment:
Here we have a third name for the same thing. So you have `scrolledToEnd`, `scrolledToBottom` and `scrolledDown`. I would recommend using `scrolledToBottom` everywhere

Reply:
Agreed, IDK why it's so inconsistent in naming 😅